superset/assets/src/visualizations/deckgl/utils.js

```diff
@@ -0,0 +1,88 @@
+import d3 from 'd3';
+import getSequentialSchemeRegistry from '../../modules/colors/SequentialSchemeRegistrySingleton';
+import { colorScalerFactory } from '../../modules/colors';
+
+export function getBreakPoints(fd, features) {
+  if (fd.break_points === undefined || fd.break_points.length === 0) {
+    // compute evenly distributed break points based on number of buckets
+    const numBuckets = fd.num_buckets
+      ? parseInt(fd.num_buckets, 10)
+      : 10;
+    const [minValue, maxValue] = d3.extent(features, d => d[fd.metric]);
+    const delta = (maxValue - minValue) / numBuckets;
+    const precision = delta === 0
+      ? 0
+      : Math.max(0, Math.ceil(Math.log10(1 / delta)));
+    return Array(numBuckets + 1)
+      .fill()
+      .map((_, i) => (minValue + i * delta).toFixed(precision));
+  }
+  return fd.break_points.sort((a, b) => parseFloat(a) - parseFloat(b));
+}
+
+export function hexToRGB(hex, alpha = 255) {
```

Comment:
There is already a `hexToRGB` function in `colors.js`. 
Why create a new copy here?

Reply:
I think it got duplicated because of refactoring. Will remove this one.

Reply:
It seems like only deck.gl components are using `hexToRGB` so you can consider moving it here too. I may try to refactor and deprecate `colors.js` in the near term.